packages/neo-one-smart-contract-test-common/src/withContracts.ts

```diff
@@ -105,8 +113,18 @@ export const withContracts = async <T>(
   test: (contracts: T & TestOptions) => Promise<void>,
   createCompilerHost: () => CompilerHost,
   getDataProvider: () => Promise<DataProviderOptions>,
-  { ignoreWarnings = false, deploy = true, autoConsensus = true, autoSystemFee = true }: WithContractsOptions = {},
+  {
+    ignoreWarnings = false,
+    deploy = true,
+    autoConsensus = true,
+    autoSystemFee = true,
+    logging = false,
+  }: WithContractsOptions = {},
 ): Promise<void> => {
+  if (!logging) {
+    disableConsoleLogForTest();
```

Comment:
I think we want to leave these enabled - otherwise users don't have a good way to debug using `console.log` in their smart contracts.